### PR TITLE
Created landing spot for centrality and similarity algorithms

### LINKED ISSUE #3023: [DOC]: Create cuGraph Algorithm landing page and template for individual Algorithms
### How would you describe the priority of this documentation request

Medium

### Describe the future/missing documentation

CuGraph docs that explain the available algorithms and their support level and use.
Create a template page explaining how each algorithm will be documented.
o	Example code
o	Use case for what the algorithm does
o	Academic reference
o	API Link to cuGraph implementation
o	Support level (MG, directed, pylibcugraph, datatypes)

### Where have you looked?

https://github.com/rapidsai/cugraph/tree/branch-22.12/notebooks/algorithms

### Code of Conduct

- [X] I agree to follow cuGraph's Code of Conduct
- [X] I have searched the [open documentation issues](https://github.com/rapidsai/cugraph/issues?q=is%3Aopen+is%3Aissue+label%3Adoc) and have found no duplicates for this bug report

### LINKED ISSUE #3030: [DOC]: Add first Algorithms to new structure
### How would you describe the priority of this documentation request

Critical (currently preventing usage)

### Describe the future/missing documentation

Add community and centrality algorithms to new algo sections.

### Where have you looked?

https://github.com/rapidsai/cugraph/tree/branch-23.02/notebooks/algorithms

### Code of Conduct

- [X] I agree to follow cuGraph's Code of Conduct
- [X] I have searched the [open documentation issues](https://github.com/rapidsai/cugraph/issues?q=is%3Aopen+is%3Aissue+label%3Adoc) and have found no duplicates for this bug report

### PR DESCRIPTION
**Pull request by acostadon**
resolves #3030 
resolves #3023

### CODE REVIEW

File: docs/cugraph/source/graph_support/algorithms/betweenness_centrality.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Betweenness Centrality
+
+Betweenness centrality is a measure of the relative importance based on measuring the number of shortest paths that pass through each vertex or over each edge.  High betweenness centrality vertices have a greater number of path cross through the vertex.  Likewise, high centrality edges have more shortest paths that pass over the edge.
```
Comment (BradReesWork) [inline docs/cugraph/source/graph_support/algorithms/betweenness_centrality.md:3]: you should introduce the short hand here:  "Between Centrality (BC) ..."

File: docs/cugraph/source/graph_support/algorithms/jaccard_similarity.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Jaccard Similarity
+
+The Jaccard similarity between two sets is defined as the ratio of the volume of their intersection divided by the volume of their union. 
+
+The Jaccard Similarity can then be defined as
+
+Jaccard similarity coefficient = |A ∩ B| / |A ∪ B|
```
Comment (BradReesWork) [inline docs/cugraph/source/graph_support/algorithms/jaccard_similarity.md:7]: use the math tool to make equations better

File: docs/cugraph/source/graph_support/algorithms/katz_centrality.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Katz Centrality
+
+Katz centrality is a measure of the relative importance of a vertex within the graph based on measuring the influence across the total number of walks between vertex pairs. Katz is similar to Eigenvector centrality. The main difference is that Katz also takes into account indirect relationships. The Katz calculation includes a user-controlled attenuation variable that controls the weight of indirect relationships. Otherwise it shares many of the advantages and disadvantages of Eigenvector centrality.
+
+<img src="https://latex.codecogs.com/gif.latex?C_{katz}(i)&space;=&space;\sum_{k=1}^{\infty}&space;\sum_{j=1}^{n}&space;\alpha&space;^k(A^k)_{ji}" title="C_{katz}(i) = \sum_{k=1}^{\infty} \sum_{j=1}^{n} \alpha ^k(A^k)_{ji}" />
```
Comment (BradReesWork) [inline docs/cugraph/source/graph_support/algorithms/katz_centrality.md:5]: use the same syntax you did with Sorenson - looks cleaner than the gif produced from latex.codecogs

File: docs/cugraph/source/graph_support/algorithms/Centrality.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+
+# cuGraph Centrality Notebooks
+
+<img src="../../images/zachary_graph_centrality.png" width="35%"/>
+
+cuGraph Centrality notebooks contain a collection of Jupyter Notebooks that demonstrate algorithms to identify and quantify importance of vertices to the structure of the graph.  In the diagram above, the highlighted vertices are highly important and are likely answers to questions like:
```
[the remark this answers is not in the document]
Comment (acostadon) [reply docs/cugraph/source/graph_support/algorithms/Centrality.md]: done

File: docs/cugraph/source/graph_support/algorithms/betweenness_centrality.md (hunk 1 above)
Comment (acostadon) [reply docs/cugraph/source/graph_support/algorithms/betweenness_centrality.md:3]: Done

File: docs/cugraph/source/graph_support/algorithms/katz_centrality.md (hunk 3 above)
Comment (acostadon) [reply docs/cugraph/source/graph_support/algorithms/katz_centrality.md]: fixed

File: docs/cugraph/source/graph_support/algorithms/jaccard_similarity.md (hunk 2 above)
Comment (acostadon) [reply docs/cugraph/source/graph_support/algorithms/jaccard_similarity.md]: done

File: docs/cugraph/source/graph_support/algorithms/Similarity.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+Manipulation of the data before or after the graph analytic is not covered here.   Extended, more problem focused, notebooks are being created and available https://github.com/rapidsai/notebooks-extended
+
+## Summary
+
+|Algorithm          |Notebooks Containing                                                     |Description                                                  |
+| --------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+|[Jaccard Smiliarity](./jaccard_similarity.md)| [Jaccard Similarity](https://github.com/rapidsai/cugraph/blob/main/notebooks/algorithms/link_prediction/Jaccard-Similarity.ipynb)                 ||
+|[Overlap Similarity](./overlap_similarity.md)| [Overlap Similarity](https://github.com/rapidsai/cugraph/blob/main/notebooks/algorithms/link_prediction/Overlap-Similarity.ipynb)                    ||
+|[Sorensen](./sorensen_coefficient.md)|Coming Soon||
```
[the remark this answers is not in the document]
Comment (acostadon) [reply docs/cugraph/source/graph_support/algorithms/Similarity.md:20]: Done. One last thing to correct in the notebook.

File: docs/cugraph/source/graph_support/algorithms.md [hunk 6]
```diff
@@ -20,9 +20,9 @@ _Italic_ algorithms are planned for future releases.
...
 | ----------------- | ---------------------------------- | ------------------- | --------------------------------------------------------------- |
-| [Centrality](https://github.com/rapidsai/cugraph/blob/main/notebooks/algorithms/centrality/Centrality.ipynb)        |                                    |                     |                                                                 |
+| [Centrality](./algorithms/Centrality.md)        | [Centrality](https://github.com/rapidsai/cugraph/blob/main/notebooks/algorithms/centrality/Centrality.ipynb)                                   |                     |                                                                 |
 |                   | [Katz](https://github.com/rapidsai/cugraph/blob/main/notebooks/algorithms/centrality/Katz.ipynb)                               | __Multi-GPU__ |                                                                 |
 |                   | [Betweenness Centrality](https://github.com/rapidsai/cugraph/blob/main/notebooks/algorithms/centrality/Betweenness.ipynb)             | Single-GPU          | MG planned for 23.02                                            |
```
[the remark this answers is not in the document]
Comment (acostadon) [reply docs/cugraph/source/graph_support/algorithms.md]: fixed

**Comment by acostadon** [comment]
Blocked by issue #3708
Until that is fixed the jaccard and newly created sorenson notebooks don't return correct results.

**Review by BradReesWork — CHANGES_REQUESTED** [review]
copyright and minor edits - see slack

**Pull request merged**